randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -1019,7 +1043,12 @@ Extra - asset_id: 100
   > Door to Noob Bridge
       Any of the following:
           Space Jump
+          # Freeze enemies to use as platforms
           Hi-Jump Boots and Ice Beam
+          # WJ with Red Tower Tweaks
```

Comment:
AFAIK the red tower tweaks were for the platform at the very top, near hellway

Reply:
I rolled a seed to verify. Red Tower tweaks are for the bottom. They let you get back up after going down so you don't softlock.
![image](https://github.com/randovania/randovania/assets/46821362/f9f17382-aa58-41d8-9121-72b3fb1f931c)
![image](https://github.com/randovania/randovania/assets/46821362/cbfa68d9-9dd1-487b-b6b9-bdc9e62b5a9a)